Test case for bug 37781: [XHR] Cross-Origin asynchronous request with credential raises NETWORK_ERR

PASSED

